packages/dht/src/connection/WebRTC/WebRtcConnector.ts

```diff
@@ -128,7 +130,15 @@ export class WebRtcConnector implements IWebRtcConnectorService {
         )
 
         connection.on('localCandidate', (candidate: string, mid: string) => {
-            remoteConnector.sendIceCandidate(this.ownPeerDescriptor!, candidate, mid, connection.connectionId.toString())
+            const parsedCandidate = new ParsedLocalCandidate(candidate)
+            if (this.config.externalIp && parsedCandidate.getType() === CandidateType.HOST) {
+                parsedCandidate.setIp(this.config.externalIp)
+                const injectedCandidate = parsedCandidate.toString()
+                logger.error(`onLocalCandidate injected external ip ${injectedCandidate} ${mid}`)
+                remoteConnector.sendIceCandidate(this.ownPeerDescriptor!, injectedCandidate, mid, connection.connectionId.toString())
+            } else {
+                remoteConnector.sendIceCandidate(this.ownPeerDescriptor!, candidate, mid, connection.connectionId.toString())
+            }
```

Comment:
Code repetition here, both if and else statements should end with the same call to (with only the 2nd argument varying) `remoteConnector.sendIceCandidate`.